legacy-docs/cops_rails.md

```diff
@@ -2600,27 +2601,27 @@ false positives.
 
 ```ruby
 # bad
-Model.pluck(:id).uniq
+Model.pluck(:id).distinct
 
 # good
-Model.uniq.pluck(:id)
+Model.distinct.pluck(:id)
 ```
 #### EnforcedStyle: aggressive
 
 ```ruby
 # bad
 # this will return a Relation that pluck is called on
-Model.where(cond: true).pluck(:id).uniq
+Model.where(cond: true).pluck(:id).distinct
 
 # bad
 # an association on an instance will return a CollectionProxy
-instance.assoc.pluck(:id).uniq
+instance.assoc.pluck(:id).distinct
 
 # bad
-Model.pluck(:id).uniq
+Model.pluck(:id).distinct
 
 # good
-Model.uniq.pluck(:id)
+Model.distinct.pluck(:id)
```

Comment:
The document has been changed from Markdown (legacy-docs/cops_rails.md) to AsciiDoc (docs/modules/ROOT/pages/cops_rails.adoc). So, this Markdown document is not maintained.
Can you revert this Markdown doc?

Reply:
oops, not sure how this doc got updated 🤔